### PR TITLE
WFLY-10492 Upgrade Infinispan to 9.3.1.Final

### PR DESCRIPTION
**Pull request by pferraro**
https://issues.jboss.org/browse/WFLY-10492

### CODE REVIEW

File: clustering/infinispan/extension/pom.xml [hunk 1]
```diff
@@ -90,6 +90,10 @@
             <groupId>net.jcip</groupId>
             <artifactId>jcip-annotations</artifactId>
         </dependency>
+        <dependency>
+            <artifactId>rxjava</artifactId>
+            <groupId>io.reactivex.rxjava2</groupId>
```
Comment (jamezp) [inline clustering/infinispan/extension/pom.xml]: This is a bit nitpicky, but these elements should be reversed to be consistent with the rest of the POM.

File: pom.xml [hunk 2]
```diff
@@ -223,8 +223,8 @@
         <version.gnu.getopt>1.0.13</version.gnu.getopt>
         <version.groovy-all>2.4.7</version.groovy-all>
         <version.httpunit>1.7.2</version.httpunit>
-        <version.io.netty>4.1.9.Final</version.io.netty>
-        <version.io.reactivex.rxjava>2.1.9</version.io.reactivex.rxjava>
+        <version.io.netty>4.1.25.Final</version.io.netty>
```
Comment (jamezp) [inline pom.xml]: @jmesnil Is this okay for messaging? I would imagine so since it's a micro.

File: clustering/infinispan/extension/pom.xml (hunk 1 above)
Comment (pferraro) [reply clustering/infinispan/extension/pom.xml]: Hmm.  Looks like these lines got swapped during a merge. Will fix.

File: pom.xml (hunk 2 above)
Comment (pferraro) [reply pom.xml]: @jamezp The feature branch for the artemis upgrade already uses 4.1.24.Final.

Comment (jamezp) [reply pom.xml]: Perfect thanks! I probably should have just checked :)

**Review by jamezp — APPROVED** [review]

File: pom.xml (hunk 2 above)
Comment (jmesnil) [reply pom.xml]: +1

**Pull request merged**